apps/admin-x-activitypub/src/views/Explore/Explore.tsx

```
@@ -74,7 +74,7 @@ export const ExploreProfile: React.FC<ExploreProfileProps & {
                 {profile.bio &&
                     <div
                         dangerouslySetInnerHTML={{__html: profile.bio}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)